lib/box/enter.go

```diff
@@ -33,34 +33,44 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func CombinedOutput(c libcontainer.Container, cfg ProcessConfig) ([]byte, error) {
-	var b bytes.Buffer
-	cfg.Out = &b
-	err := StartProcess(c, cfg)
-	if err != nil {
-		return b.Bytes(), err
-	}
-	return b.Bytes(), nil
+// CombinedOutput executes the command specified with cfg and returns
+// its output as a combination of stdout/stderr
+func (b *Box) CombinedOutput(cfg ProcessConfig) (output []byte, err error) {
+	var buf bytes.Buffer
+	cfg.Out = &buf
+	err = b.StartProcess(cfg)
+	return buf.Bytes(), trace.Wrap(err)
 }
 
-func StartProcess(c libcontainer.Container, cfg ProcessConfig) error {
-	log.Infof("StartProcess(%v)", cfg)
-	defer log.Infof("StartProcess(%v) is done!", cfg)
+// StartPtocess starts the command specified with cfg.
+// It does not wait for command to finish - use Box.Wait() to achieve this
```

Comment:
nit: Does not seem correct. Looks like both `startProcessTTY` and `startProcessStdout` wait for the process to finish.